StripeUICore/StripeUICoreTests/Unit/Elements/DropdownFieldElementTest.swift

```diff
@@ -93,4 +93,28 @@ final class DropdownFieldElementTest: XCTestCase {
         element.didFinish(element.pickerFieldView, shouldAutoAdvance: true)
         XCTAssertEqual(index, 0)
     }
+    
+    func testCantSelectDisabledItem() {
+        let disabledItem = DropdownFieldElement.DropdownItem(pickerDisplayName: "Disabled",
+                                                             labelDisplayName: "Disabled",
+                                                             accessibilityValue: "Disabled",
+                                                             rawData: "Disabled",
+                                                             isDisabled: true)
+        let itemsWithDisabled = items + [disabledItem]
+        
```

Comment:
Seems like we should assert the number of items before directly accessing specific indexes.